client/gutenboarding/style.scss

```diff
@@ -1,11 +1,58 @@
 @import './gutenberg-styles/styles';
 
+// Override core variables: we don't have wp-admin header and sidebar in Calypso
+$admin-bar-height: 0;
+$admin-bar-height-big: 0;
+$admin-sidebar-width: 0;
+$admin-sidebar-width-big: 0;
+$admin-sidebar-width-collapsed: 0;
```

Comment:
Would be redundant after moving to a new entrypoint in #37071?

Reply:
Nope, this is still relevant since things in `edit-post` package make assumption that it's living in `wp-admin` with the header & sidebar taking up space.